src/components/JobCard/JobCard.tsx

```diff
@@ -0,0 +1,107 @@
+        }
+    }
+
+    return (
+        <Stack spacing={1} alignItems="center">
+            <Card sx={{ bgcolor: '#512DA8', color: 'white', width: '100%' }}>
+                <CardContent>
+                    <Grid container spacing={1} alignItems="center">
+                        <Grid item xs={12} sm={12} md={2} display="flex" justifyContent="center">
+                            <Avatar
+                                sx={{ bgcolor: 'grey', width: 80, height: 80, borderRadius: 2 }}
+                                alt={`${company} logo`}
+                                src={logoUrl}
+                            />
+                        </Grid>
+                        <Grid item xs={12} sm={12} md={8}>
+                            <Typography variant="h6" component="div" sx={{ textAlign: 'left' }}>
+                                {title}
+                            </Typography>
+                            <Grid container spacing={2}>
+                                <Grid item xs={12} sm={4} sx={{ display: 'flex', justifyContent: { xs: 'flex-start', sm: 'center' }, alignItems: 'center' }}>
+                                    <BusinessIcon fontSize="small" />
+                                    <Typography fontSize="medium" sx={{ ml: 1 }}>{company}</Typography>
+                                </Grid>
+                                <Grid item xs={12} sm={4} sx={{ display: 'flex', justifyContent: { xs: 'flex-start', sm: 'center' }, alignItems: 'center' }}>
+                                    <LocationOnIcon fontSize="small" />
+                                    <Typography fontSize="medium" sx={{ ml: 1 }}>{location}</Typography>
+                                </Grid>
+                                <Grid item xs={12} sm={4} sx={{ display: 'flex', justifyContent: { xs: 'flex-start', sm: 'center' }, alignItems: 'center' }}>
+                                    <AccountBalanceWalletIcon fontSize="small" />
+                                    <Typography fontSize="medium" sx={{ ml: 1 }}>{salary}</Typography>
+                                </Grid>
+                            </Grid>
+                            <Grid container spacing={1} mt={0.5} justifyContent="flex-start">
+                                {tags.map((tag, index) => (
+                                    <Grid item key={index}>
+                                        <Chip label={tag} sx={{ bgcolor: '#6A1B9A', color: 'white', height: 32, borderRadius: '8px' }} />
+                                    </Grid>
+                                ))}
+                            </Grid>
+                        </Grid>
+                        <Grid item xs={12} sm={12} md={2} sx={{ order: { xs: -1, sm: -1, md:0 }, alignSelf: { sm: 'flex-start' } }} display="flex">
+                            <Grid container spacing={1} justifyContent="right">
+                                <Grid item>
+                                    <AccessTimeIcon fontSize="small" />
```

Comment:
```suggestion
                                    <AccessTimeIcon />
```

Let's stick with the default element sizes as defined on the [theme](http://localhost:5173/#/theme) page. This will help us stay consistent across the UI.